exchanges/kraken/kraken.go

```diff
@@ -37,6 +38,8 @@ const (
 // Kraken is the overarching type across the kraken package
 type Kraken struct {
 	exchange.Base
+	wsAuthToken string
+	wsAuthMu    sync.RWMutex
```

Comment:
🐒 We don't normally call these `*Mu`
```suggestion
	wsAuthMutex    sync.RWMutex
```